package chapter10;

import java.util.Scanner;

public class TryCatchFinallyRuntimeExceptionExample {
	public static void main(String[] args) {
					
//		int value1 = 0;
//		int value2 = 0;
		Scanner scanner = new Scanner(System.in);
		
		while (true) {
		try {
			System.out.print("첫번째 숫자를 입력하세요>");
			String data1 = scanner.nextLine();
			System.out.print("두번째 숫자를 입력하세요>");
			String data2 = scanner.nextLine();
			
			int value1 = Integer.parseInt(data1);
			int value2 = Integer.parseInt(data2);
			int result = value1 + value2;
			System.out.printf("%d + %d = %d\n", value1, value2, result);
			break;
		} catch (NumberFormatException e) {	//드래그 + alt shift z
//			e.printStackTrace();
			System.out.println("숫자를 입력하세요.");
			}
		
 		}
		
		scanner.close();
	}
}
